garrysmod/lua/includes/modules/properties.lua

```diff
@@ -111,7 +111,7 @@ end
 function GetHovered( eyepos, eyevec )
 
 	local filter = { LocalPlayer():GetViewEntity() }
-	if ( LocalPlayer():GetViewEntity() == LocalPlayer() && IsValid( LocalPlayer():GetVehicle() ) && !LocalPlayer():GetVehicle():GetThirdPersonMode() ) then table.insert( filter, LocalPlayer():GetVehicle() ) end
+	if ( LocalPlayer():GetViewEntity() == LocalPlayer() && IsValid( LocalPlayer():GetVehicle() ) && LocalPlayer():GetVehicle():IsVehicle() && !LocalPlayer():GetVehicle():GetThirdPersonMode() ) then table.insert( filter, LocalPlayer():GetVehicle() ) end

```

Comment:
You can remove the ``IsValid( LocalPlayer():GetVehicle() )`` check since ``IsVehicle`` returns false for ``NULL`` entities as opposed to erroring.

Reply:
gonna make my own change to it if i can figure it out.